backend/tests/exploregens.spec.ts

```diff
@@ -318,22 +323,33 @@ describe('exploregens unit test', () => {
         });
 
         it("location undefined", () => {
-            workspaceConfigMock.expects("get").withExactArgs(ExploreGens["INSTALLATION_LOCATION"]).returns();
+            workspaceConfigMock.expects("get").withExactArgs(ExploreGens["INSTALLATION_LOCATION"]).returns("");
```

Comment:
it is better to remove .returns("") or .returns() if you do not use the returned value